README.md

```diff
@@ -15,43 +14,63 @@ And on on occasion, I used TDD methodology to implement some tasks, such as head
 To ensure code quality and conformity an end to end (e2e) Cypress test suite was provided.
 I implemented the integragration tests using React Testing-Library to meet the project's acceptance criteria (AC).
 
+![](./images/homePage.png)
+![](./images/searchPage.png)
+![](./images/loading.png)
+![](./images/posts.png)
+
 ## Tech Stack used:
-1. React/React Hooks
-2. Styled-components
-3. Cypress
-4. React Testing-Library
+1. **React/React Hooks**
+2. **Styled-components**
+3. **Cypress**
+4. **React Testing-Library**
 
 ## Tools
-1. ClickUp
-2. Zeplin
-3. Perfect Pixel
-4. GitHub
-5. Slack
-
-## I Proposed Added Features
+1. [ClickUp](https://clickup.com/)
+2. [Zeplin](https://zeplin.io/)
+3. [Perfect Pixel](https://chrome.google.com/webstore/detail/perfectpixel-by-welldonec/dkaagdgjmgdmbnecmcefdhjekcoceebi?hl=en)
+4. **GitHub** version control repository
+5. **Slack** Team communication tool
+
+## I Proposed these Added Features
+For me this is the most important part of the project where I was able to apply the knoledge learnt to propose features and deliver solutions to improve on the app.
 In addition to the original AC, I proposed and implemented these features:
 
-1. highlights for cells containing posts with deleted author
-2. display the posts table in modal mode
-3. choose the heatmap color theme
+1. **highlights for cells** containing posts with deleted author
+2. display the posts table in **modal mode to avoid the table to show below the fold**.
+we could implement auto-scoll but the will need to scroll up back to the heatmap again. I think offering an modal display option was practical.
+3. choose the heatmap color theme provides the user with another viewing comfort
+4. make the app responsive on all devices
+5. provide integration and end to end tests for these features while retaining the original AC.
+
+## Features Solutions screenshots
+
+1. Cell highlight to indicate it contains deleted author
+![Feature #1](./images/feature#1.png)
+2. choose the heatmap color scheme
+![Feature #2](./images/feature#2.png)
+3. display the posts table in a modal window
+![Feature #3](./images/postsModal.png)
+4. responsive on small screens
+![Feature #4](./images/responsive.png)
+5. tests
+![Feature tests](./images/tests.png)
 
 [![NPM Version][npm-image]][npm-url]
 [![Build Status][travis-image]][travis-url]
 [![Downloads Stats][npm-downloads]][npm-url]
```

Comment:
The links and images don't work. Have a look [here](https://github.com/ooloo-io/reddit-timer-bahobab/blob/e71a37e8d5ca3b7f24be66748abe7770201d9201/README.md)